app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesPresenter.java

```diff
@@ -112,7 +112,6 @@ private void listenToModelChanges() {
 
         attendeeListener.getNotifier()
             .filter(attendeeModelChange -> attendeeModelChange.getAction().equals(BaseModel.Action.UPDATE))
-            .distinctUntilChanged()
```

Comment:
but adding `distinctUntilChanged` is causing issue in updates. Without it, it is working fine. Please do test on device.